File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let mul = num_list.reduce((a, b) => (a * b));
    let sum = num_list.reduce((a, b) => (a + b));
    // 배열 안 원소를 다루는 것은 reduce 메서드 사용
    if (mul < sum*sum) {
        return 1
    } else return 0
}

File: Java/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int multiple = 1;
        for (int i = 0; i < num_list.length; i++){
            multiple *= num_list[i];
        }

        int sum_square = 0;
        for (int i = 0; i < num_list.length; i++){
            sum_square += num_list[i];
        }
        sum_square = sum_square * sum_square;

        if (multiple < sum_square){
            return 1;
        } else {
            return 0;
        }
    }
}